src/lfortran/semantics/ast_common_visitor.h

```diff
@@ -5280,6 +5280,16 @@ class CommonVisitor : public AST::BaseVisitor<Derived> {
         }
     }
 
+    void is_coarray_or_atomic(std::string intrinsic_name, const Location& loc){
+        std::vector<std::string> coarray_intrinsics, atomic_intrinsics;
```

Comment:
```suggestion
        std::vector<std::string> coarray_intrinsics;
```

I see no use of `atomic_intrinsics`, either split up current vector into two else we need to change error message.

Reply:
Oh right. I'll remove this. Thanks